.gitignore

```diff
@@ -1 +1,2 @@
 /taskcluster-worker
+coverage.out
```

Comment:
Added so that a common filename could be given to the output of the coverage report and not be included in checkin.